### PR TITLE
Emoji browserify and dynamic locale optimization

### LINKED ISSUE #369: Optimize emoji annotations and browserify
Emoji annotations can be `required` dynamically, but that should be done in a separate file which should be excluded from browserify build
See https://github.com/ezra-bible-app/ezra-bible-app/pull/226#issuecomment-830739907

### PR DESCRIPTION
**Pull request by zhuiks**
This closes #369 

External emoji library and locale files for it (emoji annotations) are ignored for browserify build (we don't need it in Cordova)

So it should be thoroughly tested on Android

### CODE REVIEW

File: app/frontend/helpers/emoji_helper.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * It should not be included in browserify build as it doesn't support dynamic requires
+ * and there is no need for emoji picker under Android or other mobile platform
+ * @module emojiHelper
+ * @category Utility
+ */
+
+const LIB_PATH = 'emoji-button/dist/';
+
+module.exports.getEmojiButtonLib = function() {
+  const { EmojiButton } = require(LIB_PATH + '/index.cjs');
```
Comment (tobias-klein) [inline app/frontend/helpers/emoji_helper.js:31]: I don't think Browserify works with variables in require statements. We had issues with that before. What Browserify does is essentially pre-load the content of the require at bundling time and essentially put it in a large string inside the JS bundle.

**Review by tobias-klein — CHANGES_REQUESTED** [review]

File: app/frontend/helpers/emoji_helper.js (hunk 1 above)
Comment (zhuiks) [reply app/frontend/helpers/emoji_helper.js:31]: The whole purpose of this PR is to move all Browserify incompatible or unneeded in Cordova code to one file and ignore it in Browserify build (see updates to `packaje.json`) 
Sorry, if I didn't communicate it clearly initially.

Comment (tobias-klein) [reply app/frontend/helpers/emoji_helper.js:31]: You're right!

**Comment by tobias-klein** [comment]
I tested:
1) Startup / No JavaScript issues
2) Switching the UI language / No JavaScript issues
3) Another startup with changed language / No JavaScript issues

**Pull request merged**